pumps/sql_aggregate_test.go

```diff
@@ -35,6 +37,59 @@ func TestSQLAggregateInit(t *testing.T) {
 
 }
 
+func TestSQLAggregateWriteData_Sharded(t *testing.T) {
+	pmp := SQLAggregatePump{}
+	cfg := make(map[string]interface{})
+	cfg["type"] = "sqlite"
+	cfg["connection_string"] = "pmp_test.db"
+	cfg["table_sharding"] = true
```

Comment:
should we also add test with Postgres or we are fine with just sqlite?

Reply:
The existing tickets were just for sqlite so I went with it. Also, there is no postgres running in ci-tests.

Reply:
In this case why to rely on database file at all. Maybe just keep connection string empty and use in-memory db?

Reply:
Not sure what you mean by in-memory db? Does our new engine have such a feature when it is empty? Also, shouldn't we test with at least one SQL database?

Reply:
Yes, if you leave it empty, SQLite switch to in-memory mode.

Reply:
Is in-memory a real SQL environment?

Reply:
I removed test file and use in-memory now.

Reply:
done